doc/lua_api.txt

```diff
@@ -3109,6 +3114,10 @@ Definition tables
         --[[
         ^ Shall place item and return the leftover itemstack
         ^ default: minetest.item_place ]]
+        can_place = func(pos, node, itemstack, placer, pointed_thing),
+        --[[
+        ^ if it exists and returns true, the on_place^ is executed
```

Comment:
it should be an arrow which is not an unicode one, which isn't readable by everyone, e.g. ↑